Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,8 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner = () => {
+    return <h1>testing CB</h1>;
```

Comment:
It's always a good idea to test components connections with simple text before moving up to rendered components.